tools/sapp/sapp/interactive.py

```diff
@@ -580,10 +589,17 @@ def issues(
                 )
                 for issue in issues
             ]
-
+            features_list = [
+                self._get_leaves_issue_instance(
+                    session, int(issue.id), SharedTextKind.FEATURE
+                )
+                for issue in issues
+            ]
             issue_strings = [
-                self._create_issue_output_string(issue, sources, sinks)
-                for issue, sources, sinks in zip(issues, sources_list, sinks_list)
+                self._create_issue_output_string(issue, sources, sinks, features)
+                for issue, sources, sinks, features in zip(
+                    issues, sources_list, sinks_list, features_list
+                )
```

Comment:
Feel free to ignore, but this could be a good application of [`starmap`](https://docs.python.org/3.7/library/itertools.html#itertools.starmap)

Reply:
It might be possible but it require more code changes. We need to construct a new iterable that consist of
 `issue, sources, sinks, features` with the same index element to do `starmap(self._create_issue_output_string, new_iterable)`. I think zip provides this and it's easier to read but let me know if this is not what you are thinking on this.

Reply:
I was thinking of preserving the `zip`, and just removing the repetition of `issue, sources, sinks, features`:
```
starmap(self._create_issue_output_string, zip(issues, sources_list, sinks_list, features_list))
```
I'll yield the point that the original way is probably easier to read, and doesn't require people to look up some obscure `itertools` function :)